### DISCUSSION TITLE
Add support for XML type in stored procedure

### DISCUSSION THREAD

**Discussion by meareal** [Azure/data-api-builder#1813] [Ideas]
Would be really nice to be able to use XML type in stored procedures

`Stored procedure sample`

![image](https://github.com/Azure/data-api-builder/assets/1301692/2194a132-babe-4e73-a190-f3899df0973c)

`dab-config.json` file

![image](https://github.com/Azure/data-api-builder/assets/1301692/b2d8631f-eb94-4b3f-a9a7-502dffc0e190)

`DAB CLI`

![image](https://github.com/Azure/data-api-builder/assets/1301692/3c96fd6a-cd90-4930-b1b4-f090b12fab24)

`DAB Version`

![image](https://github.com/Azure/data-api-builder/assets/1301692/8f610752-7dd3-4ffb-8df6-45d5d6f208e1)

**Comment by seantleonard** [comment]
Hi @meareal ,

Thank you for your suggestion! We are now tracking this with #1817.
